vmdb/app/views/shared/summary/_textual_multilabel.html.haml

```diff
@@ -11,5 +11,9 @@
     - items[:values].each do |item|
       %tr
         - item.each do |value|
-          %td
-            = value
+          - if value.kind_of?(Hash)
+            %td{:title => value[:title]}
```

Comment:
Is dropping this into the title attribute enough?

The problem with title is that you only see it in mouse over.  It means that you cannot copy/paste the value (unless you view the source of the page and track it down in the HTML).  Should it be in something more substantial, like a popup div, or something?

@dclarizio, @h-kataria, @abonas, thoughts?

If everyone's fine with the title attribute, then ignore me.  :smile: